src/vm/jitinterface.cpp

```diff
@@ -9953,8 +9953,8 @@ void* CEEInfo::getPInvokeUnmanagedTarget(CORINFO_METHOD_HANDLE method,
 
 /*********************************************************************/
     // return address of fixup area for late-bound N/Direct calls.
-void* CEEInfo::getAddressOfPInvokeFixup(CORINFO_METHOD_HANDLE method,
-                                        void **ppIndirection)
```

Comment:
> These two methods return different entrypoints

Can you break down the different entrypoints that would be returned by the two methods for a P/Invoke? In other words, given some P/Invoke P, what is the difference between a direct call to the result of `getFunctionEntryPoint(P)` and the result of `getAddressOfPInvokeFixup(P)`?